CHANGELOG.md

```diff
@@ -1,50 +1,71 @@
-# Changelog
+# Changelog Number Verification API
 
-## v0.4.0-wip [Work in Progress]
 
-## Please note:
+## Table of contents
 
-- This is *work-in-progress* version, it should be considered as a draft
-- There are bug fixes to be expected and incompatible changes in upcoming versions 
-- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments
-- Version numbers before v0.4.0-wip were used during the development of this version but not released
+- **[r1.1](#r11)**
```

Comment:
We need here to mention also the previous initial release 0.3.1. First as the for r1.1 only the changes since 0.3.1 are mentioned (which is ok), second as otherwise the claim that a "Previous public-release was certified" can't be justified.

Might be that the following would work here:
```suggestion
- **[r1.1](#r11)**
- [v0.3.1](#v031)
```

The content for v0.3.1 can be copied out of the release description of v0.3.1, see https://github.com/camaraproject/NumberVerification/releases/tag/v0.3.1